.github/workflows/android_deploy_production_to_playstore.yml

```diff
@@ -0,0 +1,60 @@
+name: Android - Deploy Production build to Play Store
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_deploy_android:
+    name: Build & Deploy Android
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Generate new project
+        run: make run PACKAGE_NAME=co.nimblehq.flutter.template PROJECT_NAME=flutter_templates APP_NAME="Flutter Templates"
+
+      - name: Setup Java JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+
+      - name: Setup Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
+
+      - name: Get flutter dependencies
```

Comment:
```suggestion
      - name: Get Flutter dependencies
```

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/163/commits/6a3bbce19c9ed30ba7b8ae32b7ebce5416977ba3